models/models.articles.js

```diff
@@ -0,0 +1,11 @@
+const db = require("../db/connection.js");
+
+exports.selectArticles = () => {
+  const SQL = `SELECT articles.article_id, articles.author, articles.body, articles.created_at, title, topic, articles.votes, COUNT(comments.article_id)::int AS comment_count FROM articles LEFT JOIN comments ON comments.article_id =articles.article_id 
```

Comment:
Here as well, body should be excluded, please see Trello ticket No. 4.

Reply:
ok thanks